style.css

```diff
@@ -262,53 +275,18 @@ section.works {
   display: flex;
   align-items: center;
   background: #f7f7f9;
-  border-radius: 8px;
   border-radius: 0.53rem;
   gap: 0.66rem;
 }
 
-@media screen and (min-width: 375px) {
-  section.header {
-    padding-top: 1.562rem;
-  }
-
-  div.headline {
-    background-image: url("/images/Header-bg@2x.png");
-    padding: 18% 8%;
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-position: left;
-  }
-
-  .nav-links a {
-    text-decoration: none;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 1rem;
-    line-height: 1.333rem;
-    color: #344563;
-  }
-
-  h1 {
-    color: #091e42;
-  }
-
-  .headline-alexandra_small {
-    display: none;
-  }
-
-  .headline-alexandra_big {
-    display: block;
-  }
-}
-
```

Comment:
- [ ] To fix the background image you remove this part which was especially for the mobile version. For this reason I can't see any background images in the mobile version. Kindly fix this issue.

![image](https://user-images.githubusercontent.com/60083437/227485483-6c02d3ad-6ffd-4fc7-a621-6fc7e5d0d64e.png)

Reply:
@ragangithub, I got your email. In the review, I meant that 
- I can't see your background image for the mobile version because you remove those lines of code I marked. 
- Kindly add those lines of code, fix the `size and position of the background image` as per the previous reviewer's suggestion, and check if you can see the background image that matches the given design.